### PR TITLE
Updated README.md to provide instructions for running the SPipes Dock…

### PR DESCRIPTION
**Pull request by palagdan**
Resolves https://github.com/kbss-cvut/s-pipes/issues/238.

…er container without mounting directories, and added optional steps for mounting directories on both Linux and Windows.

### CODE REVIEW

**Comment by blcham** [comment]
I will review during weekend

File: README.md [hunk 1]
```diff
@@ -85,17 +85,65 @@ The following software needs to be installed on the system for development:
...
+
+SPipes web can be run and exposed at port 8080 with the following command:
+
+`docker run -p 8080:8080 s-pipes-engine:latest` 
+
+To mount a directory from your host machine to the Docker container, use the following command:
+`docker run -v C:\path\on\host:/path/in/container -p 8080:8080 s-pipes-engine:latest`
+
+*Example Usage*:
+`docker run -v C:\Users\[YourUsername]\Documents:/home -p 8080:8080 s-pipes-engine:latest`
```
Comment (blcham) [inline README.md:129]: We need to discuss this at the next meeting. This mount would work but would be useless as it is not an alternative to:
> The `-v /home:/home` option mount your home to docker image - this is very convenient for testing.

Comment (blcham) [reply README.md:129]: Probably the easiest way to fix it is to not have it for Windows at all.

**Review by blcham — CHANGES_REQUESTED** [review]
I have one comment to discuss. After resolution I would make the description more concise -- i.e. have one guide with steps and differentiate only within steps (regarding Windows and Linux)

File: README.md (hunk 1 above)
Comment (palagdan) [reply README.md:129]: When I was working on this issue, I thought that the user could mount any directory on their local machine.

Comment (blcham) [reply README.md:129]: No, I believe the point here was to align "host" paths to "s-pipes-engine docker image" paths, i.e.:
- if I would have a script in `/home/blcha/projects/s-pipes/doc/example/hello-world.sms.ttl` and an error occurs in loading the script within the "docker image", I could use the path from the log file generated by "s-pipes docker image" in my "host"

Comment (blcham) [reply README.md:129]: To have the same functionality I think you would have to mount something from the link that i provided you here:
https://github.com/kbss-cvut/s-pipes/issues/238#issuecomment-2220430925

To be more precise I guess this is relevant part of the link:
```
if running docker-compose.exe or spe.bat in windows (not docker in wsl distribution) add the :
SHARED_ROOT=/host_mnt/c
PWD=/host_mnt/$PROJECT_ROOT should be absolute path
if running docker-compose or spe.sh in wsl distribution, set variables as follows:
SHARED_ROOT=/mnt/c
```

Comment (palagdan) [reply README.md:129]: This section is only appliable for wsl distributions : /host_mnt/c is a path in wsl. I don't know how to apply it to windows FS.

**Comment by blcham** [comment]
@palagdan I reformulated a little what you wrote; now I consider it final ... let me know what u think

**Comment by palagdan** [comment]
> @palagdan I reformulated a little what you wrote; now I consider it final ... let me know what u think

Thank you for the update. I agree with your changes and consider it final as well.

**Pull request merged**